code/__DEFINES/maths.dm

```diff
@@ -30,11 +30,7 @@
 // round() acts like floor(x, 1) by default but can't handle other values
 #define FLOOR(x, y) ( round((x) / (y)) * (y) )
 
-#if DM_VERSION < 513
-#define CLAMP(CLVALUE,CLMIN,CLMAX) ( max( (CLMIN), min((CLVALUE), (CLMAX)) ) )
-#else
 #define CLAMP(CLVALUE,CLMIN,CLMAX) clamp(CLVALUE, CLMIN, CLMAX)
```

Comment:
is this really needed anymore? just rename all instances of `CLAMP` to `clamp` imo